app-frontend/src/app/pages/projects/edit/color/color.html

```diff
@@ -17,28 +17,28 @@ <h5 class="sidebar-title">Color Correction</h5>
     </li>
   </ul>
 </div>
+
 <div class="sidebar-scrollable">
-  <ul class="sidebar-list">
-    <li class="separator">
-      <button class="btn btn-light btn-block"
-              disabled>
-        Auto Tone
-      </button>
-      <i class="icon-info"></i>
-    </li>
-    <li class="separator">
-      <button class="btn btn-light btn-block"
-              disabled>
-        Auto Contrast
-      </button>
-      <i class="icon-info"></i>
-    </li>
-    <li class="separator">
-      <button class="btn btn-light btn-block"
-              disabled>
-        Auto Color
-      </button>
-      <i class="icon-info"></i>
-    </li>
-  </ul>
+  <div class="list-group">
+    <div class="list-group-item">
+      <div style="flex-direction: row;">
```

Comment:
This style declaration is unneeded. While the inherited flex direction of this element is `column` and this does change it to `row`, there is only one child element, and as a result this really has no effect.